packages/vitest/src/node/git.ts

```diff
@@ -12,7 +12,7 @@ export class VitestGit {
   constructor(private cwd: string) {}
 
   private async resolveFilesWithGitCommand(args: string[]): Promise<string[]> {
-    let result: ExecaReturnValue
+    let result: Result
```

Comment:
This should narrow doens the type of `result.stdout` below to `string|undefined`:

```suggestion
    let result: Result<{}>
```